ansible/roles/debops.tinyproxy/defaults/main.yml

```diff
@@ -0,0 +1,501 @@
+---
+# .. vim: foldmarker=[[[,]]]:foldmethod=marker
+
+# debops.tinyproxy default variables
+# ==================================
+
+# .. contents:: Sections
+#    :local:
+#
+# .. include:: ../../../../includes/global.rst
+
+
+# Installation, APT packages [[[
+# ------------------------------
+
+# .. envvar:: tinyproxy__base_packages [[[
+#
+# List of APT base packages which are required by the Tinyproxy application.
+tinyproxy__base_packages: [ 'tinyproxy' ]
+
+                                                                   # ]]]
+# .. envvar:: tinyproxy__packages [[[
+#
+# List of APT packages which are required by the Tinyproxy application.
+tinyproxy__packages: []
+
+                                                                   # ]]]
+# .. envvar:: tinyproxy__allow [[[
+#
+# List of IP addresses or CIDR subnets which will be allowed to connect to the
+# Tinyproxy server in :command:`ip(6)tables` and TCP wrappers. If it's empty, remote
+# connections are not allowed.
+tinyproxy__allow: []
+
+                                                                   # ]]]
+# .. envvar:: tinyproxy__port [[[
+#
+# Port number on which this Tinyproxy listens on.
+tinyproxy__port: '8888'
+
+                                                                   # ]]]
+# Configuration for other Ansible roles [[[
+# -----------------------------------------
+
+# .. envvar:: tinyproxy__ferm__dependent_rules [[[
+#
+# Configuration for :ref:`debops.ferm` Tinyproxy role.
+tinyproxy__ferm__dependent_rules:
+
+  - type: 'accept'
+    dport: [ 'tinyproxy' ]
+    saddr: '{{ tinyproxy__allow }}'
+    accept_any: False
+    weight: '40'
+    role: 'tinyproxy'
+
+                                                                   # ]]]
+# .. envvar:: tinyproxy__etc_services__dependent_list [[[
+#
+# Configuration for the :ref:`debops.etc_services` Ansible role.
+tinyproxy__etc_services__dependent_list:
+
+  - name: 'tinyproxy'
+    port: '{{ tinyproxy__port }}'
+
+                                                                   # ]]]
+# Tinyproxy configuration file [[[
+# --------------------------------
+
+# The variables below define the contents of the
+# :file:`/etc/tinyproxy/tinyproxy.conf` configuration file.
```

Comment:
You could add there something like:
```
See :ref:`tinyproxy__ref_configuration` for more details.
```
so that in the documentation, users have a handy link to the reference page for these variables.